community/elasticsearch/APKBUILD

```diff
@@ -19,6 +20,7 @@ source="https://artifacts.elastic.co/downloads/$pkgname/$pkgname-$pkgver.tar.gz
 builddir="$srcdir/$pkgname-$pkgver"
 
 default_module=transport-netty4
+mkdir -p $builddir/modules
 _modules=$(find $builddir/modules -mindepth 1 -maxdepth 1 -type d ! -name $default_module -exec basename {} \;)
```

Comment:
Yeah, it’s horrible. I didn’t want this to be merged. Actually I’m still thinking about reverting it. :/ @itoffshore basically threw the cleanup around init script I did in previous version away…